Blog-Andrei-Dona.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <title>
+        Blog
+    </title>
+
+    <div style="position:fixed;top:0px;height: 130px;background-color: blanchedalmond;">
+        <img src="https://i.pinimg.com/originals/3f/78/08/3f7808a419c74cd1eec609ce71b9fced.jpg"
+            style="position: relative;width: 15%;top: 0px"></img>
+        <span style="position: absolute;top: 0px;font-size: 130px;">
+            Andrei's Blog</span>
+    </div>
+
+</head>
+
+<body style="background-color: seashell;height: 2000px;">

```

Comment:
Incearca sa pui toate stilurile separat, in tag-ul `style`. In felul acesta te obisnuiesti si cu folosirea selectorilor, si este mult mai usor de urmarit/modificat.